generated/nirfmxnr/nirfmxnr.proto

```diff
@@ -733,11 +733,6 @@ enum AcpSweepTimeAuto {
   ACP_SWEEP_TIME_AUTO_TRUE = 1;
 }
 
```

Comment:
Note: Bluetoooth and wlan are out-of-sync with scrapigen so I didn't import those.

Reply:
lets make sure to throw a task on those user stories for Api cleanup